src/reducers/books.js

```diff
@@ -0,0 +1,16 @@
+const bookReducer = (state = null, action) => {
```

Comment:
Now!, we need to combine this `book reducers` inside `src/reducers/index.js`
![image](https://user-images.githubusercontent.com/31897434/76904510-68c4b780-68a0-11ea-996b-975e870cd081.png)

**Read the documentation!** 

Example: 
```
combineReducers({
  books,
});
```

Reply:
Corrections done,
reducers/index.js uses now combineReducers method to add books.js reducer